R/tm_g_lineplot.R

```diff
@@ -518,21 +518,51 @@ srv_g_lineplot <- function(id,
   checkmate::assert_class(data, "tdata")
 
   shiny::moduleServer(id, function(input, output, session) {
-    anl_inputs <- teal.transform::merge_expression_module(
-      datasets = data,
+    selector_list <- teal.transform::data_extract_multiple_srv(
       data_extract = list(x = x, y = y, strata = strata, paramcd = paramcd, y_unit = y_unit, param = param),
+      datasets = data,
+      select_validation_rule = list(
+        x = shinyvalidate::sv_required("Please select a single time variable"),
+        y = shinyvalidate::sv_required("Please select a single analysis variable"),
```

Comment:
```suggestion
        x = shinyvalidate::sv_required("Please select a time variable"),
        y = shinyvalidate::sv_required("Please select an analysis variable"),
```

I think these only allow one selection anyway as cs_to_des_select for x and y have multiple = FALSE